src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -82,101 +116,81 @@ export default function GamesSettings({ useDetails = true }: Props) {
         </p>
       )}
 
-      {!nativeGame && (
+      <Tabs value={value} onChange={handleChange} aria-label="settings tabs">
+        {!isWin && !nativeGame && <Tab label="Wine" value="wine" />}
+        <Tab label={t('settings.navbar.other', 'Other')} value="other" />
+        {!isCrossover && !isWin && (
+          <Tab
+            label={t('settings.navbar.advanced', 'Advanced')}
+            value="advanced"
+          />
+        )}
+        {hasCloudSaves && (
+          <Tab
+            label={t('settings.navbar.sync', 'Cloud Saves Sync')}
+            value="saves"
+          />
+        )}
+      </Tabs>
+
+      <TabPanel value={value} index={'wine'}>
```

Comment:
we can remove the fragment after this line now

Reply:
fixed